### ISSUE TITLE
No Data when Zabbix's last data point extends prior to Time Picker Time 

### ISSUE THREAD

**Issue by mrosun** [alexanderzobnin/grafana-zabbix#561]
### I'm submitting a ...
- [X] Bug report
- [ ] Feature requestdd
- [ ] Question / Support request

_If you don't send bug report, remove sections below_
### For bug report please include this information:

**Software Versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 5.0.4   | 3.4.8  | 3.9.0                 |

#### What did you do?
Created a Single Stat Panel.
Data Query to a point with Interval 1 day. 
Set Grafana time picker to last 6 hours.

#### What was the expected result?
Show the latest data.

#### What happened instead?
Received N/A.

If I select time picker to last 24 hours, the correct data shows up.

If it relates to metric data visualization would be great to get
- An image or text representation of your metric query
![problem 6](https://user-images.githubusercontent.com/319488/38767449-e9b5e8f2-3fa6-11e8-89bb-bac7091bc2b1.jpg)
![problem 5](https://user-images.githubusercontent.com/319488/38767450-ee769c88-3fa6-11e8-9950-b2a8d40abcc8.jpg)
![problem 4](https://user-images.githubusercontent.com/319488/38767451-f194c85e-3fa6-11e8-8b54-176ea07ebab4.jpg)
![problem 3](https://user-images.githubusercontent.com/319488/38767452-f752776e-3fa6-11e8-9b9c-7a75a1a22098.jpg)
![problem 1](https://user-images.githubusercontent.com/319488/38767454-fc5e3c7a-3fa6-11e8-8fb7-cda8530d8309.jpg)
![problem 2](https://user-images.githubusercontent.com/319488/38767455-fdccce0a-3fa6-11e8-9eed-0ae6a69b5b89.jpg)

- The raw query and response for the network request (check this in chrome dev tools network tab, here you can see metric requests and other request, please include the request body and request response)

Data OK 
Response:
{"jsonrpc":"2.0","result":[{"itemid":"52426","clock":"1523630026","value":"8","ns":"742750875"}],"id":1}

No Data 
Response:
{"jsonrpc":"2.0","result":[],"id":1}

**Comment by mrosun** [comment]
I would prefer not to set the time range to 1 or 2 days.

That is a solution that I am using at the moment.

I was hoping current value = last value, but I see that it does not.

**Comment by alexanderzobnin** [comment]
No, current value is last value in the dataset. But dataset is empty for 6 hours.
